app/build.gradle

```diff
@@ -17,8 +17,8 @@ android {
   defaultConfig {
     minSdkVersion deps.android.build.minSdkVersion
     targetSdkVersion deps.android.build.targetSdkVersion
-    versionCode 3010
-    versionName "3.0.1"
+    versionCode 3100
```

Comment:
Please revert these changes. (Most probably you used them to test an upgrade?).

Reply:
oops yes